### DISCUSSION TITLE
TODO

### DISCUSSION THREAD

**Discussion by Jimut123** [Jimut123/jimutmap#19] [Ideas]
#### Here are the list of todos **(Open an issue if you want to contribute!)**:

- [ ] Stitch tiles by specifying length and breadth of tiles. 
- [ ] Get roads route in json format from specified latitude longitudes, this can help the data scientists to create optimal routes from point A to B.
- [ ] Convert all the class of objects present in the map to JSON data via image processing, which can help the geoscientists to process information at a large scale.
- [ ] Find all the unique items that can be present in the map. (Like caffe icon, shop icon, temple icon etc.)
- [ ] Fetch all the necessary objects (shops/caffe etc.) that are queried to in a certain radius of coordinate.
- [ ] Probably use other 3D data to create 3D world from satellite 2D imagery.
- [ ] Create a conda package

**Comment by Akashchatterj** [comment]
>> Probably use other 3D data to create 3D world from satellite 2D imagery.
Will be tough

**Reply by Jimut123** [reply]
Need to find other allied datasets. Please report if you find one.

**Reply by Jimut123** [reply]
Also, can use separate issue for this.

**Comment by felipeffm** [comment]
Why routes from https://www.openstreetmap.org/ are not enough? Their graph of roads does not satisfy the route optimization problem?
